cloudify_cli/commands/teardown.py

```diff
@@ -139,15 +127,12 @@ def _assert_force(force):
             "command should be executed.")
 
 
-def _do_teardown(task_retries, task_retry_interval, task_thread_pool_size):
+def _do_teardown():
     # reload settings since the provider context maybe changed
     p = env.get_profile_context()
     provider_context = p.provider_context
     bs.read_manager_deployment_dump_if_needed(
         provider_context.get('cloudify', {}).get('manager_deployment'))
-    bs.teardown(
-        task_retries=task_retries,
-        task_retry_interval=task_retry_interval,
-        task_thread_pool_size=task_thread_pool_size)
+    bs.teardown()
```

Comment:
What about the call to bs? Don't you need to clean it as well? Is it the same repo?